src/tools/sfdk/commandlineparser.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "commandlineparser.h"
 
+#include <sfdk/sdk.h>
```

Comment:
Should come after local includes, separated from other includes with blank lines.